docs/modules/ROOT/pages/type-definitions/schema-configuration/type-configuration.adoc

```diff
@@ -0,0 +1,234 @@
+[[schema-configuration-type-configuration]]
+= Type Configuration
+
+A GraphQL Object type, representing a Neo4j Node, produces multiple operation fields in the `Query`, `Mutation`, and `Subscription` types, for example:
+
+[source, graphql, indent=0]
+----
+type Movie { 
+    title: String 
+    length: Int
+}
+----
+
+From this type definitions, the library generates the following operation fields:
```

Comment:
I'm not sure if we should use singular or plural form here. I've seen the same expression before, but thought about this only now. 

```suggestion
From these type definitions, the library generates the following operation fields:
```

or 

```suggestion
From this type definition, the library generates the following operation fields:
```